src/components/custom-sdk/template/HMRC_ODX_GDSCheckAnswersPage/index.tsx

```diff
@@ -82,20 +82,29 @@ export default function HmrcOdxGdsCheckAnswersPage(props: HmrcOdxGdsCheckAnswers
       });
   }
 
+  useEffect(() => {
+    if (sessionStorage.getItem('isComingFromEduStartPages') === 'true') {
+      sessionStorage.setItem('isEditMode', 'true');
+      sessionStorage.setItem('isComingFromPortal', 'false');
+      sessionStorage.setItem('isComingFromEduStartPages', 'false');
+    }
+  });
```

Comment:
```suggestion
  }, []);
```
Don't forget to add the dependency array